service/tasks/driver.py

```diff
@@ -32,7 +32,7 @@
 from core.models.identity import Identity
 from core.models.profile import UserProfile
 
-from service.deploy import check_process, wrap_script, echo_test_script,\
+from service.deploy import _inject_env_script, check_process, wrap_script, echo_test_script,\
```

Comment:
Maybe I'm missing the mark on the purpose of `_method_name` - but when I see an underbar method imported - that's a signal for me to pay extra attention. I believe that with `_` preview we're saying that `_inject_env_script` is _intended_ to be use *within* `service.deploy` - but not outside. 

I know we need to forge ahead ... or [_ford the river_](http://coldfrontmag.com/wp-content/uploads/2013/06/2-Trail_069.jpg). But just something I _note_ as I look at code. 

Let me know if I'm off-base.